spec/features/comments_spec.rb

```diff
@@ -1,19 +1,20 @@
 require 'rails_helper'
 
-Rspec.describe "comments", type: :feature do
+RSpec.describe "comments", type: :feature do
   describe "Adding a comment" do
```

Comment:
You'll also need a spec for if the form fails validations, the user should get an error message if they don't enter commenter or body